zkevm-circuits/src/tx_circuit.rs

```diff
@@ -189,6 +195,8 @@ impl<F: Field> SubCircuitConfig<F> for TxCircuitConfig<F> {
         // tag, rlp_tag, tx_type, is_none
         let tx_type = meta.advice_column();
         let rlp_tag = meta.advice_column();
+        let tx_value_rlc = meta.advice_column();
```

Comment:
is this phase2?

Reply:
fixed in https://github.com/scroll-tech/zkevm-circuits/pull/719/commits/d7a0eabb6f05fe2de1e3007693c578928974f56e.